### PR TITLE
Create 2017-11-20-convocatoria-editor

### PR DESCRIPTION
**Pull request by mariajoafana**
NOTICE: Are you attempting to submit a lesson to the Programming Historian? We no longer accept lesson submissions by pull request to this repository. Please consult our [guidelines for submitting a lesson](http://programminghistorian.org/new-lesson-workflow) for further instructions.

If your pull request instead concerns some issue with the formatting or functioning of our site, or some error in an existing lesson, please see our technical contribution guidelines: https://github.com/programminghistorian/jekyll/wiki/Making-technical-contributions

### CODE REVIEW

File: _posts/2017-11-20-convocatoria-editor.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Convocatoria para editor de contenidos en español de The Programming Historian 
+authors:
+- Maria Jose Afanador-Llach
+layout: post
+categories: posts
+---
+
+Desde su conformación el (equipo editorial de contenidos en español)[https://programminghistorian.org/posts/announcing-new-team-spanish-language-editors], ha traducido (28 tutoriales)[https://programminghistorian.org/es/lecciones/] del inglés al español con el apoyo de una red de colaboradores de países tales como Argentina, España, Colombia y México. Continuando con nuestro compromiso con la diversidad lingüística, The Programming Historian está buscando un nuevo miembro para contribuir al equipo de editores de contenidos en español. El editor o editora trabajará para hacer accesibles los tutoriales de PH al público hispanoparlante, facilitará los procesos de traducción de tutoriales y actuará como editor de los mismos. De igual forma apoyará las estrategias para construir una comunidad ayude a aumentar el acceso y la colaboración de PH en el mundo hispanoparlante.
```
Comment (mdlincoln) [inline _posts/2017-11-20-convocatoria-editor.md]: Fix all links so they follow the pattern [link text](link url)

**Review by mdlincoln — CHANGES_REQUESTED** [review]

File: _posts/2017-11-20-convocatoria-editor.md (hunk 1 above)
Comment (mariajoafana) [reply _posts/2017-11-20-convocatoria-editor.md]: Thanks, done!

**Review by mdlincoln — APPROVED** [review]

**Comment by mdlincoln** [comment]
OK, everything looks like it's showing up as intended now. Remember to add the `.md` suffix to the end of filenames for any lessons or posts in the future.

I think the text sounds great, but I'll let @acrymble and others take a look before merging.

**Comment by mariajoafana** [comment]
Are we ready to publish this?

**Comment by acrymble** [comment]
Fine with me.

**Pull request merged**